core/shared/src/main/scala/laika/parse/code/languages/HaskellSyntax.scala

```diff
@@ -0,0 +1,43 @@
+package laika.parse.code.languages
```

Comment:
Like with the other PR's, it would be better to add the license header to all new class files (and an author tag to the class scaladoc, if you wish).